pyproject.toml

```diff
@@ -87,6 +87,56 @@ known-first-party= ["pint"]
 
 
 [tool.ruff]
+extend-select = [
+#    "F",           # pyflakes
```

Comment:
I like how you included all options here, such that one can slowly work towards following all these rules 😄